types/node/scripts/generate-docs/tsconfig.json

```
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    "target": "esnext",
```

```suggestion
    "target": "es2018",
```
or similar, to remove this error:
```bash
> @ run C:\develop\DefinitelyTyped\types\node\scripts\generate-docs
> ts-node docs.ts

C:\develop\DefinitelyTyped\types\node\scripts\generate-docs\node-doc-processing.ts:45
        module.methods ??= [];
                       ^^^

SyntaxError: Unexpected token '??='
```
https://github.com/TypeStrong/ts-node/issues/903